python/graphstorm/gconstruct/remap_result.py

```diff
@@ -461,67 +572,102 @@ def main(args, gs_config_args):
+                        "r",  encoding='utf-8') as f:
+                    info = json.load(f)
+                    ntypes = info["emb_name"]
+                    emb_ntypes = ntypes if isinstance(ntypes, list) else [ntypes]
+
+        else: # There is no shared file system
+            emb_names = os.listdir(node_emb_dir)
+            emb_names = [e_name for e_name in emb_names if e_name != "emb_info.json"]
 
-    # TODO remap embeddings
+            emb_ntypes = emb_names
 
     ################## remap prediction #############
-    # if pred_etypes (edges with prediction results)
-    # is not empty, we need to remap edge prediction results.
-    # Note: For distributed SageMaker runs, pred_etypes must be
-    # provided if edge prediction result remap is required,
-    # as result_info.json is only saved by rank0 and
-    # there is no shared file system.
-    if len(pred_etypes) > 0:
-        for pred_etype in pred_etypes:
-            assert os.path.exists(os.path.join(predict_dir, "_".join(pred_etype))), \
-                f"prediction results of {pred_etype} do not exists."
-
-    # If pred_ntypes (nodes with prediction results)
-    # is not empty, we need to remap edge prediction results
-    # Note: For distributed SageMaker runs, pred_ntypes must be
-    # provided if node prediction result remap is required,
-    # as result_info.json is only saved by rank0 and
-    # there is no shared file system.
-    if len(pred_ntypes) > 0:
-        for pred_ntype in pred_ntypes:
-            assert os.path.exists(os.path.join(predict_dir, pred_ntype)), \
-                f"prediction results of {pred_ntype} do not exists."
-
-    if with_shared_fs:
-        # Only when shared file system is avaliable,
-        # we will check result_info.json for
-        # pred_etypes and pred_ntypes.
-        # If shared file system is not avaliable
-        # result_info.json is not guaranteed to exist
-        # on each instances. So users must use
-        # --pred-ntypes or --pred-etypes instead.
-        #
-        # In case when both --pred-ntypes or --pred-etypes
-        # are provided while result_info.json is also avaliable,
-        # GraphStorm remaping will follow --pred-ntypes or --pred-etypes
-        # and ignore the result_info.json.
-        if os.path.exists(os.path.join(predict_dir, "result_info.json")):
-            # User does not provide pred_etypes.
-            # Try to get it from saved prediction config.
-            with open(os.path.join(predict_dir, "result_info.json"),
-                        "r",  encoding='utf-8') as f:
-                info = json.load(f)
-                if len(pred_etypes) == 0:
-                    pred_etypes = [list(etype) for etype in info["etypes"]] \
-                        if "etypes" in info else []
-                if len(pred_ntypes) == 0:
-                    pred_ntypes = info["ntypes"] if "ntypes" in info else []
+    if predict_dir is not None:
```

Comment:
i suppose this is just the modification of indent?

Reply:
Yes.